apis/python/src/tiledbsoma/_query.py

```diff
@@ -427,6 +431,9 @@ def to_anndata(
             drop_levels:
                 If true, drop unused categories from the ``obs`` and ``var`` dataframes.
                 Defaults to ``False``.
+            dask:
+                If not ``None``, load the X layer as a dask array. See
```

Comment:
Nit: I believe `Dask` is capitalized when used in this manner, e.g., "... as a Dask array".

Reply:
Need to mark this new interface as `experimental` (lifecycle tag)